Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "c2pa-c"
-version = "0.3.0"
+version = "0.4.0"
 edition = "2021"
 authors = ["Gavin Peacock <gpeacock@adobe.com"]
 
 [lib]
-crate-type = ["lib","cdylib"]
+crate-type = ["lib","cdylib", 'staticlib']
```

Comment:
Nitpick, likely?
```suggestion
crate-type = ["lib","cdylib", "staticlib"]
```

Reply:
Actually I don't think I need to build the static Rust library here. I had been experimenting with that.